taskana-adapter-camunda-spring-boot-test/src/test/java/pro/taskana/adapter/integration/ArchitectureTest.java

```diff
@@ -0,0 +1,41 @@
+package pro.taskana.adapter.integration;
+
+import static com.tngtech.archunit.core.domain.properties.CanBeAnnotated.Predicates.annotatedWith;
+import static com.tngtech.archunit.lang.conditions.ArchPredicates.are;
+import static com.tngtech.archunit.lang.syntax.ArchRuleDefinition.methods;
+
+import com.tngtech.archunit.core.domain.JavaClasses;
+import com.tngtech.archunit.core.importer.ClassFileImporter;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestFactory;
+import org.junit.jupiter.api.TestTemplate;
+
+class ArchitectureTest {
+
+  private static JavaClasses importedClasses;
+
+  @BeforeAll
+  static void init() {
+    // time intensive operation should only be done once
+    importedClasses = new ClassFileImporter().importPackages("pro.taskana");
```

Comment:
Please import the `accaptance` package aswell, since we have 2 Files which are in an `acaptance..` package.